support/templates/nvidia-device-plugin.yaml

```
@@ -0,0 +1,50 @@
+{{ if .Values.config.nvidiaDevicePlugin.azure.enabled }}
```

```suggestion
{{- if .Values.config.nvidiaDevicePlugin.azure.enabled -}}
```

This is not at all important change, in practice it will mean this file won't start out with a blank line which is non-problematic, but I find it useful to be consistent with a practice so one doesn't end up needing to know when it will be or wont be important.

My rule of thumb is to:

1. Always chomp the whitespace to the left
2. When its the first content of a yaml file, or a named template defined in a _helpers.tpl file, also chomp right

---

Is "chomp" the hyphen? I don't know what this word means in this context! (I agree with the suggestion though)

---

Yepp, it means that whitespace left or right within the helm template (but not whitespace generated by the helm template) should be trimmed on the left or right side.

Chomping/trimming left doesn't do anything here since there were no whitespace before `{{- if`, but trimming right will make sure that the whitespace (a new line) following `...enabled -}}` is removed, so that the rendered k8s resource manifest starts with `# ...` rather than `(new line)# ...` for example.

For some helm documentation about this, see https://helm.sh/docs/chart_template_guide/control_structures/

---

Thanks! I applied your suggestion in [`49082fb`](https://github.com/2i2c-org/infrastructure/pull/932/commits/49082fbd0beaaf0d097e84f53b027b129c322790)